lib/bitfields/version.rb

```diff
@@ -1,3 +1,3 @@
 module Bitfields
-  Version = VERSION = "0.9.0"
+  Version = VERSION = "0.9.1"
```

Comment:
don't bump in PRs ... also this is not a patch, but a minor (new feature)

Reply:
Sorry!